src/index.js

```diff
@@ -0,0 +1,117 @@
+import angular from 'angular';
+
+import ChatbotCtrl from './controller';
+import ChatbotServiceProvider from './service';
+import directive from './directive';
+
+import iconsTemplate from './templates/icons.html';
+import embeddedTemplate from './templates/embedded.html';
+
+import './index.less';
+
+const moduleName = 'ngOvhChatbot';
+
+angular
+  .module(moduleName, [
+    'ngAnimate',
+    'ngEmbed',
+    'ovh-angular-user-pref',
+  ])
+  .controller('chatbotController', ChatbotCtrl)
+  .provider('chatbotService', ChatbotServiceProvider)
+  .directive('ovhChatbot', directive)
+  .run(($templateCache) => {
+    $templateCache.put('ng-ovh-chatbot-icons-template.html', iconsTemplate);
+    $templateCache.put('ng-ovh-chatbot-embedded-template.html', embeddedTemplate);
+  })
+  .constant('CHATBOT_MESSAGE_TYPES', {
```

Comment:
Both `CHATBOT_MESSAGE_TYPES` and `CHATBOT_NGEMBED_OPTIONS` could be extracted to a `constants.js` file.